Reneiski_Homework1/ReneiskiSelenium11/Program.cs

```diff
@@ -0,0 +1,239 @@
+﻿using NUnit.Framework;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+using OpenQA.Selenium.Interactions;
+using OpenQA.Selenium.Support.UI;
+
+namespace Reneiski_Homework1
+{
+    public class Selenium
+    {
+        private IWebDriver _driver;        
+        private WebDriverWait _driverWait;
+        private Actions _driverActions;
+
+        [OneTimeSetUp]
+        public void OneTimeSetup()
+        {
+            _driver = new ChromeDriver();
+            _driver.Manage().Window.Maximize();
+            _driverWait = new WebDriverWait(_driver, TimeSpan.FromSeconds(1000));
+            _driverActions = new Actions(_driver);
+        }
+
+        [Test]
+        public void FirstCheckBoxTest()
+        {
+            _driver.Navigate().GoToUrl("https://demoqa.com/checkbox");
+
+            var firstCheckBox = _driver.FindElement(By.XPath("//*[@class='rct-icon rct-icon-uncheck']"));
+            firstCheckBox.Click();
+
+            var checkedCheckbox = _driver.FindElement(By.XPath("//*[@class='rct-icon rct-icon-check']"));
+            var isCheckboxChecked = checkedCheckbox.GetAttribute("class").Contains("check");
+            
+            Assert.IsTrue(isCheckboxChecked);
+
+            var currentResult = _driver.FindElement(By.Id("result"));
+            var currentResultText = currentResult.Text;
+            
+            Assert.AreEqual(currentResultText, ("You have selected :\r\nhome\r\ndesktop\r\nnotes\r\ncommands\r\ndocuments\r\nworkspace\r\nreact\r\nangular\r\nveu\r\noffice\r\npublic\r\nprivate\r\nclassified\r\ngeneral\r\ndownloads\r\nwordFile\r\nexcelFile"));
```

Comment:
you have the expected result as the actual result
![image](https://user-images.githubusercontent.com/55654931/209945085-98ec1a3c-5add-4833-b3ba-2ed39e922d2e.png)

Reply:
+ you can create separate private const string with expected result on the test class level and put into Assert this string. Code will be more readable

Reply:
please correct across the file

Reply:
Fixed